connaisseur/validators/static/static_validator.py

```diff
@@ -0,0 +1,20 @@
+from connaisseur.validators.interface import ValidatorInterface
+from connaisseur.image import Image
+from connaisseur.exceptions import ValidationError
+import logging
+
+
+class StaticValidator(ValidatorInterface):
+
+    name: str
+    approve: bool
+
+    def __init__(self, name: str, approve: bool, **kwargs):
+        self.name = name
```

Comment:
With the suggested change in the parent class, `super().__init__(name, **kwargs)`